styles.css

```diff
@@ -294,3 +298,62 @@ li {
   background-color: white;
   border-radius: 0.25rem;
 }
+
+/* Contact Section */
+.contact-section {
+  margin: 0 auto;
+  padding: 2.5rem 1rem 4rem;
+  text-align: center;
+}
```

Comment:
- [ ] [OPTIONAL] Great design, but I think it can kill even more even you implement the later part where you have a great deal of `margin` or `padding` to separate the social icons from the last element of your contact section as shown 👇 

![image](https://user-images.githubusercontent.com/99715640/213027007-ca42438c-ddfc-4214-bc4e-5110876e2a39.png)

Reply:
Hi @AbayomiOlaoye ,

Thank you for the approval!
I'm also going to apply the suggested optional change to make it more user-friendly 😎 Thanks!